### PR TITLE
Add usage example of evm-lite-lib

### PR DESCRIPTION
**Pull request by junwei0117**
evm-lite-lib has been officially released today ( 01/11 ) , 
we will continue to track the library and update our sample code
let SmartContract deployment and interaction more faster.

### CODE REVIEW

**Comment by jserv** [comment]
Can you do `rebase` and `push --force`? Thus, we can eliminate the commits of `Merge branch ...`.

**Comment by junwei0117** [comment]
As mentioned, I've done, please check it again.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "Deploy environment for running Lightweight Ethereum node with interchangeable consensus.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DLTcollab/evm-lite-env.git"
+  },
+  "author": "BiiLabs",
```
Comment (jserv) [inline package.json]: The author would be "BiiLabs, Co. Ltd.`".

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/DLTcollab/evm-lite-env/issues"
+  },
+  "homepage": "https://github.com/DLTcollab/evm-lite-env#readme",
```
Comment (jserv) [inline package.json]: Is the anchor `#readme` required?

**Comment by jserv** [comment]
Add new dedicated `README.md` in directory `example`, which summarizes each example contract.

File: example/deploySmartContractLocally.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const fs = require('fs');
+
+// Default from address
+const from = '0x1a5c6b111e883d920fd24fee0bafae838958fa05';
+
+// Config evm-lite IP
+const nodeIP = '127.0.0.1';
+
+// Data directory object
+const dataDirectory = new evmlib.DataDirectory('/Users/junwei.evmlc');
```
Comment (jserv) [inline example/deploySmartContractLocally.js]: Why would you hardcode data directory?

File: example/deploySmartContractLocally.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+// Password for keystore
+const password = 'superpassword';
```
Comment (jserv) [inline example/deploySmartContractLocally.js]: Mention the account information in specific `README.md`.

File: example/deploySmartContractLocally.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+// Contract object
+const contractPath = '/Users/junwei/Documents/CrowdFunding.sol';
```
Comment (jserv) [inline example/deploySmartContractLocally.js]: Ditto. No hardcode path is allowed.

File: example/transferKeystoreLocally.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+const to = '0xeF5D1019e5B752E993BA17EC3aD2B021872Fb969';
+
+// Transaction Value
+const value = 200;
+
+// Config evm-lite IP
+const nodeIP = '127.0.0.1';
+
+// Data directory object
+const dataDirectory = new evmlib.DataDirectory('/Users/junwei.evmlc');
```
Comment (jserv) [inline example/transferKeystoreLocally.js]: Ditto.

**Comment by jserv** [comment]
I am thinking of the indentation tools and/or style guide that we should follow. Any suggestions?

File: package.json (hunk 1 above)
Comment (junwei0117) [reply package.json]: I have corrected the naming more rigorous.

**Comment by junwei0117** [comment]
In my case, I am trying to follow the `Airbnb JavaScript Style Guide` with `ESLint` using `VScode`.
There are still a lot of mainstream guide, e.g. `Google`, `GitHub`, `JavaScript Standard`.
And `ESLint` is a pluggable linting utility for JavaScript.

File: package.json (hunk 2 above)
Comment (junwei0117) [reply package.json]: No, I have removed it.

File: example/transferKeystoreLocally.js (hunk 6 above)
Comment (junwei0117) [reply example/transferKeystoreLocally.js]: I have replaced it with simple instructions.

File: README.md [hunk 7]
```diff
@@ -25,6 +25,70 @@ Parameters :
...
+
+As well as the following description, you can take a look at our examples folder for more.
+
+### Transfer
+
+1. Init the transaction addresses.
+   ```javascript
+    const from = '[from address]';
+    const to = '[to address]';
+    //const from = '0x479e8b1b9d8b509755677f6d61d2f7339ba4c0fd';
```
Comment (jserv) [inline README.md:38]: Can these example read certain configurations from specific file(s)? Therefore, we can eliminate such hardcode paths.

**Comment by jserv** [comment]
@mushroom0117, Please send another pull request which enforces style check for consistency.

File: README.md (hunk 7 above)
Comment (junwei0117) [reply README.md:38]: The addresses are randomly generated by `evm-lite wallet`.
`evm-lite wallet` path can be modified by the user.

Comment (jserv) [reply README.md:38]: File `example/README.md` should mention the approach to configure these contracts as well.

File: example/transferKeystoreLocally.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
+const evmlib = require('evm-lite-lib');
+
+// Transaction Addresses
+const from = '[from address]';
+const to = '[to address]';
+const value = '[value]';
+
+// EVMLC object
+const evmlc = new evmlib.EVMLC('127.0.0.1', 8080, {
```
Comment (jserv) [inline example/transferKeystoreLocally.js:9]: Can you avoid preconfigured hostname and port number by fetching them via specific config file?

File: package.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "evm-lite-env",
+  "version": "0.1.0",
+  "description": "Deploy environment for running Lightweight Ethereum node with interchangeable consensus.",
```
Comment (jserv) [inline package.json]: It is not Ethereum node anymore. Instead, we would call it EVM node since it is now decoupled from Ethereum ecosystem.

Comment (junwei0117) [reply package.json]: Okay, I have made the word more precise in a new commit.

**Pull request merged**